tiler/app/utils/cors_middleware.py

```diff
@@ -0,0 +1,26 @@
+from fastapi.middleware.cors import CORSMiddleware
+from os import getenv
+
+def add_cors_middleware(app):
+  cors_origins_regex = getenv("NETWORK_CORS_ORIGINS_REGEX", "")
```

Comment:
I need to understand what this is doing buuut I trust you so much

Reply:
> I trust you so much

and you shouldn't (especially for any commits that come from me after mid-afternoon when my brain may feel close to retirement) 😅 _maniacal evil laughter_

well, here we should mostly trust the Starlette developers 😅 - this bit is simply reading the new env var I suggested to use  and pass its content to the `cors_origins_regex` argument of the Starlette CorsMiddleware ((https://www.starlette.io/middleware/#corsmiddleware), which can use regular expressions to match against the `Origin` header sent by browsers.

Reply:
aha got it. Thank you!